akka-actor/src/main/scala/akka/japi/JavaAPI.scala

```diff
@@ -83,8 +82,6 @@ object Pair {
  *
  * This class is kept for compatibility, but for future API's please prefer [[akka.japi.function.Creator]].
  */
-@silent("@SerialVersionUID has no effect")
-@SerialVersionUID(1L)
```

Comment:
While indeed the compiler says this SerialVersionUID 'has no effect', removing it is not strictly speaking binary compatible: `mimaReportBinaryIssues` reports the static field does disappear. Can we be sure that doesn't cause any problems?

Reply:
The static field `public static final long serialVersionUID = 1L;` is ONLY generated by scala 2.12.
scala 2.13 don't generate that field.
See also: lampepfl/dotty#8429

Reply:
The static field `public static final long serialVersionUID = 1L;` is ONLY generated by scala 2.12.
scala 2.13 don't generate that field.
I think we be sure that doesn't cause any problems.
See also: lampepfl/dotty#8429